docs/common/ca.md

```diff
@@ -104,7 +104,64 @@ In addition to the above CAs, you can install other CAs via RPM. These only work
 Verifying CA Certificates
 -------------------------
 
-After installing the CA certificates, they can be verified with the following command:
+### Are up-to-date ###
+
+It is important to keep CA certificates up-to-date for grid services and their clients to maintain integrity of
+the production grid.
+To verify that your CA certificates are on the latest version on a given host, determine the most recently released
+versions and the method by which your CA certificates have been installed:
+
+1.  Retrieve the versions of the most recently released
+    [IGTF CA certificates](https://repo.opensciencegrid.org/cadist/index.html) and
+    [OSG CA certificates](https://repo.opensciencegrid.org/cadist/index-new.html)
+
+1.  Determine which of the three CA certificate installation methods you are using:
+
+    1.  Check for an existing RPM installation (only one of the following RPM packages may be installed at a time):
+
+            :::console
+            # rpm -q igtf-ca-certs osg-ca-certs
```

Comment:
Something along the lines of:

Run this command to find out which of the following items applies to you:
```suggestion
            # rpm -q igtf-ca-certs osg-ca-certs empty-ca-certs osg-ca-scripts
```
if `igtf-ca-certs` or `osg-ca-certs` are installed, read option 1. If `osg-ca-scripts` is installed, read option 2. If `empty-ca-certs` is installed, read option 3.